apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java

```diff
@@ -262,6 +271,14 @@ public Bundle findByPatient(
 
     List<IBaseResource> eobs = new ArrayList<IBaseResource>();
 
+    if (claimTypes.contains(ClaimTypeV2.INPATIENT)) {
+      if (lastUpdated.getLowerBound().getValue().after(DATE_OF_DEFECT)
+          && lastUpdated.getLowerBound().getValue().before(DATE_OF_FIX)) {
+        lastUpdated.setLowerBound(
+            new DateParam().setPrefix(ParamPrefixEnum.LESSTHAN_OR_EQUALS).setValue(DATE_OF_DEFECT));
+      }
+    }
+
```

Comment:
@KyRobbins Makes sense to me. cc: @christrotteradhoc

Reply:
Agreed - we will need more commenting in all of the places (this particular piece of code is very much a work in progress that is still pending some discussions within the team). As for removing this after a certain date, we are going to add it to the list of things to remove in V3 for sure but I think we were leaning more towards leaving it in V2 forever since we don't know for sure a date by which all consumers would have issued a new call. If we do have such a date though we can add that in a comment.

Reply:
Just to pose the question, was there any thought in just updating the `lastUpdated` of the affected claims rather then adding a temporary code patch?

Reply:
we are not doing this anymore but one of the main reasons to not update the DB was to allow this to be specific to v2.